.gitignore

```diff
@@ -8,7 +8,7 @@
 Makefile
 release
 .DS_Store
-build*
```

Comment:
This will start allowing shadow-build directories like `build-Qt5.12-Bill-Cool` to show up, instead of ignoring them.

Reply:
As discussed on Slack, I'll switch this over to `/build*/ to only remove build directories inside the root directory.